src/vault/entities/migrations.go

```diff
@@ -0,0 +1,11 @@
+package entities
+
+import "time"
+
+type MigrationStatus struct {
```

Comment:
In order to provide better insides about what it is the actual status you should also add the total amount of accounts or just add the pending amount which is decresing over time

Reply:
done